From b8b8a0d4dadc34087aa42a761bcc306e5043a27e Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 12 Feb 2018 13:24:45 +0800
Subject: [PATCH 3360/5242] MLK-17590-01 driver: clk: imx: update the clk flag
 of pll

commit  d636de6fb2732372e97e7ca45f000a64bedb9778 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add CLK_GET_RATE_NOCACHE and CLK_SET_RATE_GATE for sscg pll.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-sccg-pll.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/clk/imx/clk-sccg-pll.c b/drivers/clk/imx/clk-sccg-pll.c
index 69b65e2..a871878 100644
--- a/drivers/clk/imx/clk-sccg-pll.c
+++ b/drivers/clk/imx/clk-sccg-pll.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 NXP.
+ * Copyright 2017-2018 NXP.
  *
  * The code contained herein is licensed under the GNU General Public
  * License. You may obtain a copy of the GNU General Public License
@@ -219,7 +219,7 @@ struct clk *imx_clk_sccg_pll(const char *name, const char *parent_name,
 		break;
 	}
 
-	init.flags = 0;
+	init.flags = CLK_SET_RATE_GATE | CLK_GET_RATE_NOCACHE;
 	init.parent_names = &parent_name;
 	init.num_parents = 1;
 
-- 
1.7.9.5

